inference-engine/src/gna_plugin/backend/am_intel_dnn.cpp

```diff
@@ -790,8 +790,12 @@ void GNAPluginNS::backend::AMIntelDNN::WriteDnnText(const char *filename, intel_
                     uint32_t num_bytes_per_bias = component[i].op.affine.num_bytes_per_bias;
                     float weight_scale_factor = component[i].op.affine.weight_scale_factor;
                     float output_scale_factor = component[i].output_scale_factor;
+#ifdef DUMP_WB
```

Comment:
Maybe use `#if defined(DUMP_WB) || defined(LIGHT_DUMP)` in cases where the same code is wrapped to reduce the number of lines?